test/system/flows/student/mentor_finished_discussion_test.rb

```diff
@@ -48,9 +48,9 @@ class MentorFinishedDiscussionTest < ApplicationSystemTestCase
           click_on "It was good!"
           fill_in "Leave #{discussion.mentor.handle} a testimonial (optional)", with: "Good mentor!"
           click_on "Finish"
-          click_on "Back to the exercise"
+          click_on "Continue to exercise"
 
-          assert_text "Nice, it looks like you’re done here!"
+          assert_text "Want to try another mentor?"
```

Comment:
@dem4ron Why has this changed. If someone selects "It was good!", I don't think we should be taking them down the "request another mentor" path